lib/active_fedora/orders/ordered_list.rb

```diff
@@ -12,15 +12,15 @@ class OrderedList
       #   stored.
       # @param [::RDF::URI] head_subject URI of head node in list.
       # @param [::RDF::URI] tail_subject URI of tail node in list.
-      def initialize(graph, head_subject, tail_subject)
+      def initialize(graph, head_subject, tail_subject, node_cache = NodeCache.new)
```

Comment:
seems like a straightforward improvement to me. could i suggest we use a named argument, instead? this should be more tolerant toward any potential extensions (since it doesn't depend on argument order).

```suggestion
      def initialize(graph, head_subject, tail_subject, node_cache: NodeCache.new)
```

Reply:
Sure thing, change committed